lib/tasks/data_import_field.rake

```diff
@@ -20,4 +20,26 @@ namespace :core do
       raise "Field #{field} cannot be updated by data_import_field"
```

Comment:
Field #{field} cannot be updated by _lettings_data_import_field_ ? Is there any other documentation we need to update?

Reply:
Probably not